packages/react-table/src/components/Table/utils/decorators/collapsible.tsx

```diff
@@ -19,23 +25,38 @@ export const collapsible: IFormatter = (
     property
   };
 
+  const rowId = rowIndex !== undefined ? rowIndex : -1;
```

Comment:
this caused a breaking change, as now, unless rowIndex is defined, the CollapseColumn's toggle button will not render since `isOpen` will also be undefined.
To reproduce, simply don't pass in `rowIndex` for the first `<Td>` element that is for the collapse toggle button.